source/beamline/BioXAS/BioXASSideBeamline.cpp

```diff
@@ -268,7 +266,7 @@ void BioXASSideBeamline::setupComponents()
 	// Detector stage.
 
 	detectorStageLateral_ = new CLSMAXvMotor("SMTR1607-6-I22-16", "SMTR1607-6-I22-16", "SMTR1607-6-I22-16", true, 0.05, 2.0, this, ":mm");
-	connect( detectorStageLateral_, SIGNAL(connected(bool)), this, SLOT(updateConnected()) );
+	addDetectorStageLateralMotor(detectorStageLateral_);
```

Comment:
`ge32ElementDetector_` is instantiated in `BioXASSideBeamline`. Should we call function `addGe32Detector()` to add it to the set?